fidesctl/src/fidesctl/core/generate_dataset.py

```diff
@@ -11,95 +11,37 @@
 from fideslang.models import Dataset, DatasetCollection, DatasetField, Taxonomy
 from .utils import get_db_engine, echo_green, echo_red
 
+SCHEMA_EXCLUSION = {
+    "postgresql": ["information_schema"],
+    "mysql": ["mysql", "performance_schema", "sys", "information_schema"],
+    "mssql": ["INFORMATION_SCHEMA", "guest", "sys"],
+    "snowflake": ["information_schema"],
+    "redshift": ["information_schema"],
+}
 
-def get_postgres_collections_and_fields(
-    engine: Engine,
-) -> Dict[str, Dict[str, List[str]]]:
-    """
-    Postgres-specific implementation that to ingest all of the
-    schemas and tables from a database.
-
-    Returns a mapping of schemas to its tables and all of the table's fields.
-    """
-    inspector = sqlalchemy.inspect(engine)
-    schema_exclusion_list = ["information_schema"]
-
-    db_tables: Dict[str, Dict[str, List]] = {}
-    for schema in inspector.get_schema_names():
-        if schema not in schema_exclusion_list:
-            db_tables[schema] = {}
-            for table in inspector.get_table_names(schema=schema):
-                db_tables[schema][f"{schema}.{table}"] = [
```

Comment:
hi @brentonmallen1 @ThomasLaPiana, I(more like Brenton) refactored this logic so there's not so much repeated code. As part of that I changed the table name in our dataset to no longer include the schema -  `{schema}.{table}` to `{table}`. I don't think we could come up with a good reason to keep this, and it simplifies the logic a bit. If you feel that we should keep this let's discuss that here, otherwise this PR is ok to merge.

Reply:
that makes total sense to me, good call here @earmenda / @brentonmallen1

Reply:
upon further reading, this code cleanup is top-tier, looks way better now and also simplifies the testing. good work here team!